diff_cover/tests/test_violations_reporter.py

```diff
@@ -976,7 +976,162 @@ def test_quality_pregenerated_report_continuation_char(self):
         self.assertEqual(violations, [Violation(2, u"W1401: Invalid char '\ufffd'")])
 
 
+class JsHintQualityReporterTest(unittest.TestCase):
+
+    def setUp(self):
+
```

Comment:
delete this blank line. Sorry to flake8 you. Don't care about the long lines though (im looking at the diff-quality report :-P)